meta/saisanitycheck.c

```diff
@@ -4920,6 +4926,20 @@ void check_graph_connected()
             continue;
         }
 
+        if (SAI_OBJECT_TYPE_P4EXT_ENTRY == i)
```

Comment:
In the original header file from Kram, we had a [dummy reference to P4EXT entry object from the Route object](https://github.com/barefootnetworks/SAI/blob/sai_p4ext/inc/sairoute.h#L90). This kept the sai sanity check happy. But I believe that could lead to unnecessary questions and pushback during the upstreaming. So I decided to remove that reference and add this check to the sanity checker.